server/events/project_finder.go

```diff
@@ -40,6 +40,9 @@ type ProjectFinder interface {
 	DetermineProjectsViaConfig(log *logging.SimpleLogger, modifiedFiles []string, config valid.RepoCfg, absRepoDir string) ([]valid.Project, error)
 }
 
+// IgnoredFiles contains filename fragments to ignore while looking at changes
+var IgnoredFiles = [...]string{"terraform.tfstate", "terraform.tfstate.backup", "tflint.hcl"}
```

Comment:
```suggestion
var ignoredFilenameFragments = []string{"terraform.tfstate", "terraform.tfstate.backup", "tflint.hcl"}
```

lowercase so it isn't exported and while I just learned about `[...]` (cool!) I think its best to keep things consistent.